consensus/clique/src/main/java/org/hyperledger/besu/consensus/clique/jsonrpc/methods/CliqueGetSigners.java

```diff
@@ -71,7 +72,7 @@ private Optional<BlockHeader> determineBlockHeader(final JsonRpcRequestContext r
     final Optional<BlockParameter> blockParameter;
     try {
       blockParameter = request.getOptionalParameter(0, BlockParameter.class);
-    } catch (Exception e) { // TODO:replace with JsonRpcParameter.JsonRpcParameterException
+    } catch (JsonRpcParameter.JsonRpcParameterException e) {
```

Comment:
Be nicer to static import JsonRpcParameter.JsonRpcParameterException so don't have the qualified exception here

Reply:
Can do